extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -176,11 +194,28 @@ export class AmpIframe extends AMP.BaseElement {
   }
 
 
+  /**
+   * Relayouts are needed after the initial render if we don't have an iframe,
+   * since it may be removed when the doc becomes inactive.
+   * @override
+   **/
+  isRelayoutNeeded() {
```

Comment:
Just retested without this, it is definitely needed. Re: https://github.com/ampproject/amphtml/pull/2121#discussion_r53421618